packages/react-server-cli/src/cli.js

```diff
@@ -0,0 +1,44 @@
+import parseCliArgs from "./parseCliArgs"
+
+// weirdly, we parse the args twice. the first time we are just looking for --production, which
+// affects the default values for the other args.
+const isProduction = parseCliArgs(false).production || (process.env.NODE_ENV === "production"); //eslint-disable-line no-process-env
+// now if production was sent in on the command line, let's set NODE_ENV if it's unset.
+if (isProduction && !process.env.NODE_ENV) { //eslint-disable-line no-process-env
+	process.env.NODE_ENV = "production"; //eslint-disable-line no-process-env
+}
+const argv = parseCliArgs(isProduction);
+
+// these require calls are after the argument parsing because we want to set NODE_ENV
+// before they get loaded.
+const logging = require("react-server").logging,
+	logger = logging.getLogger(__LOGGER__),
+	start = require(".").start;
```

Comment:
Happy to switch to `./index` if you want. Felt slightly DRYer this way but slightly more confusing.